src/controllers/NotificationController.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Tests for Notification Controllers
+ * Send HTTP requests to Notification Controllers and check mocked responses
+ */
+
+import * as express from "express";
+import { PagoPaConfig } from "../Configuration";
+import { ControllerError } from "../enums/ControllerError";
+import { HttpErrorStatusCode } from "../enums/HttpErrorStatusCode";
+import { NotificationSubscriptionRequestType } from "../enums/NotificationSubscriptionType";
+import * as NotificationAPI from "../services/NotificationAPI";
+import { NotificationSubscriptionRequestCtrl } from "../types/controllers/NotificationSubscriptionRequestCtrl";
+import * as AppResponseConverter from "../utils/AppResponseConverter";
+import * as RestfulUtils from "../utils/RestfulUtils";
+
+export function activeNotificationsSubscription(
+  req: express.Request,
+  res: express.Response,
+  pagoPaConfig: PagoPaConfig
+): void {
+  updateNotificationsSubscription(
+    req,
+    res,
+    NotificationSubscriptionRequestType.ACTIVATION,
+    pagoPaConfig
+  ).catch();
```

Comment:
never swallow exceptions. if you want a no-op in case of any exception, just let it bubble up until the main(),
then catch the exception there (at top level) and log it